build/aws-marketplace/cloudbuild.yaml

```diff
@@ -19,6 +19,6 @@ steps:
       - '-ec'
       - |
         source ~/.heroic-ecr-secret && \
-        docker tag nakama-awsmp:${TAG#v} $_MAP_ECR_REPOSITORY:$TAG
+        docker tag nakama-awsmp:${TAG#v} $_MAP_ECR_REPOSITORY:${TAG#v}-m1
```

Comment:
What's this change?

Reply:
That's versioning of the images since we can't replace them in the mp ecr.

Reply:
I think we leave the old images as they are - and we just publish Nakama images that don't exist yet.

Reply:
Alternatively we use the `r` suffix as we've used everywhere else before that stands for `revision`. I don't know what `M` stands for here?

Reply:
Ok that's fine, let's use `-m` suffix :)